scio-extra/src/main/scala/com/spotify/scio/extra/sparkey/package.scala

```diff
@@ -124,12 +124,16 @@ package object sparkey {
       val uri = SparkeyUri(basePath, self.context.options)
       require(!uri.exists, s"Sparkey URI ${uri.basePath} already exists")
       logger.info(s"Saving as Sparkey: $uri")
+      val atLeastOneElement: SCollection[Option[(K, V)]] =
```

Comment:
this solution seems a bit hacky to me -- how would you feel about implementing, for example, a `FileBasedSink` for Sparkey output? it supports default destinations for empty PCollections so we wouldn't have to add these extra transforms to work around it. @jto - wdyt?

Reply:
Same. This solution really feels hacky.

Reply:
It is hacky!
I would also love a specialized IO for Sparkey instead of current implementation.
I'm not sure how it plays with sideInputs where Sparkey is used.